Templates/AutoModelGeneratable.stencil

```diff
@@ -1,9 +1,14 @@
 {# Defining macros for each annotation #}
 
 
+{% macro AddEntity type %}
 
```

Comment:
Remove this newline under extension. To avoid having 2 newlines after imports as it's now

Reply:
And regenerate models)